docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/bitbucket/index.md

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_label: Bitbucket
+---
+
+# The Bitbucket Webhook Receiver
+
+The Bitbucket Webhook Receiver will only respond to Repsitory pushevent as originating from
+Bitbucket repositories.
+
+In response to a `repo:push` event, the receiver "refreshes" `Warehouse's` subscribed to the Bitbucket repository from which the event originated.
+
+:::info
+"Refreshing" a `Warehouse` resource means enqueuing it for immediate
+reconciliation by the Kargo controller, which will execute the discovery of
+new artifacts from all repositories to which that `Warehouse` subscribes.
+:::
+
+## Configuring the Receiver
+
+The Bitbucket webhook receiver will need to reference a Kubernetes `Secret` with a
+`secret` key in its data map. This [shared
+secret](https://en.wikipedia.org/wiki/Shared_secret) will be used by Bitbucket to
+sign requests. The receiver will use it to authenticate those requests by
+verifying their signatures.
+
+:::note
+The following command is suggested for generating a complex secret:
+```shell
+openssl rand -base64 48 | tr -d '=+/' | head -c 32
+```
+:::
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+    name: bb-wh-secret
+    namespace: kargo-demo
+stringData:
+    secret: <your-secret-here>
+---
+apiVersion: kargo.akuity.io/v1alpha1
+kind: ProjectConfig
+metadata:
+  name: kargo-demo
+  namespace: kargo-demo
+spec:
+  webhookReceivers: 
+    - name: bb-wh-receiver
+      bitbucket:
+        secretRef:
+          name: bb-wh-secret
+```
+
+## Retrieving the Receiver's URL
+
+Kargo will generate a hard-to-guess URL from the configuration. We can obtain 
+this URL using the following command:
+
+```
+kubectl \
+    get projectconfigs \
+    kargo-demo \
+    -n kargo-demo \
+    -o=jsonpath='{.status.webhookReceivers}'
+```
+
+:::note
+If your project config has configured multiple webhook receivers, it is recommended to describe the `ProjectConfig` resource in the kargo project namespace and copy the URL. For example, 
+```
+kubectl \
+    describe projectconfig \
+    my-project-config \
+    -n kargo-project namespace
+```
+:::
+
+## Registering with Bitbucket
+
+**Step 1:** Navigate to `https://bitbucket.org/<workspace>/<repository_name>/admin/webhooks` where `<workspace>` has been replaced with your Bitbucket workspace for which you are an administrator and `repository_name` has been replaced with the name of a repository belonging to that workspace.
+
+**Step 2:** Click the <Hlt>Add webhook</Hlt> button to create a webhook for the repository.
+
+**Step 3:** On the <Hlt>Add new webhook</Hlt> page
+1. Enter a <Hlt>Title</Hlt> with a short description.
```

Comment:
```suggestion
**Step 3:** On the <Hlt>Add new webhook</Hlt> page

1. Enter a <Hlt>Title</Hlt> with a short description.
```

If you start a list, always leave a newline in between. This makes the raw Markdown much easier to read.